reana_job_controller/job_manager.py

```diff
@@ -122,7 +122,7 @@ def cache_job(self):
         """Cache a job."""
         workflow = Session.query(Workflow).filter_by(
             id_=self.workflow_uuid).one_or_none()
-        access_times = calculate_file_access_time(workflow.get_workspace())
+        access_times = calculate_file_access_time(workflow.workspace_path)
```

Comment:
We can now upgrade the minimal version of `reana-db` to `0.7.0.dev20200205` in `setup.py`.